app/src/test/java/ch/epfl/polybazaar/LiteListingAdapterTest.java

```diff
@@ -0,0 +1,38 @@
+package ch.epfl.polybazaar;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import ch.epfl.polybazaar.UI.LiteListingAdapter;
+import ch.epfl.polybazaar.litelisting.LiteListing;
+
+import static org.junit.Assert.assertEquals;
+
+public class LiteListingAdapterTest {
+
+        LiteListingAdapter liteListingAdapter;
+
+        @Before
+        public void init() {
+            LiteListing liteListing1 = new LiteListing("1", "listing1", "CHF 1");
```

Comment:
I think you shouldn't put "CHF" into the liteListing, but add it when you set the TextView, something like that : String.format("CHF %s", liteListing.getPrice()).

Reply:
yes, good idea. I implemented it.